### PR TITLE
frontend: convert `checkbackup` into a functional component.

### PR DESCRIPTION
**Pull request by shonsirsha**
To follow the latest conventions and best practices of React, we'd like to convert our class components into functional ones.

Here, we're converting `checkbackup.tsx` into a functional component.

### CODE REVIEW

File: frontends/web/src/routes/device/bitbox02/checkbackup.tsx [hunk 1]
```diff
@@ -14,107 +14,83 @@
...
-    activeDialog: false,
-    message: '',
-    foundBackup: undefined,
-    userVerified: false,
-  };
-
-  private checkBackup = async () => {
-    const { backups, deviceID, t } = this.props;
-    this.setState({ message: t('backup.check.confirmTitle') });
+  const checkBackup = async () => {
```
Comment (thisconnect) [inline frontends/web/src/routes/device/bitbox02/checkbackup.tsx:38]: tinynit: you could directly `export const checkBackup = async () => {` here

untested LGTM

**Review by thisconnect — APPROVED** [review]
LGTM with tinynit

File: frontends/web/src/routes/device/bitbox02/checkbackup.tsx (hunk 1 above)
Comment (shonsirsha) [reply frontends/web/src/routes/device/bitbox02/checkbackup.tsx:38]: you meant in the component level `export const Check = ({deviceId, ...})` right?

Comment (thisconnect) [reply frontends/web/src/routes/device/bitbox02/checkbackup.tsx:38]: yes I meant export component ofc, thank you   🙈  🙃

**Pull request merged**